incubator/patroni/templates/NOTES.txt

```diff
@@ -0,0 +1,29 @@
+Getting Started:
+
+Patroni can be accessed via port 5432 on the following DNS name from within your cluster:
+{{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local
+
+To get your password for superuser or admin run:
+
+    # admin user password
+    printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.data.password-admin[*]}"`);echo
+
+    # superuser password
+    printf $(printf '\%o' `kubectl get secret --namespace {{ .Release.Namespace }} {{ template "fullname" . }} -o jsonpath="{.data.password-superuser[*]}"`);echo
+
+To connect to your database:
+
+1. Run an Ubuntu pod that you can use as a client:
+
+    kubectl run -i --tty ubuntu --image=ubuntu:16.04 --restart=Never -- bash -il
```

Comment:
It seems to hang when when running psql directly, probably because it's waiting for a password, but you can't really enter it.

I could devide it in two steps:

```
kubectl run -i --tty --rm psql --image=postgres --command -- sh
# psql -U admin -h {{ template "fullname" . }}.{{ .Release.Namespace }}.svc.cluster.local postgres
```

What do you think of that?